data/src/main/java/io/gnosis/data/backend/TransactionSerivceApi.kt

```diff
@@ -13,7 +13,7 @@ interface TransactionServiceApi {
     @GET("v1/safes/{address}/balances/usd/")
     suspend fun loadBalances(@Path("address") address: String): List<ServiceBalance>
 
-    @GET("v1/safes/{address}/all-transactions/")
+    @GET("v1/safes/{address}/all-transactions/?limit=100")
```

Comment:
Useful for testing while we don't have paging. Must be removed, when we have paging

Reply:
Actually, I would say this should be removed before this PR is merged.